### PR TITLE
Added Evergene Presence

### PR DESCRIPTION
**Pull request by Sound-Cat**
image

image

image

### CODE REVIEW

File: websites/E/Evergene/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "thumbnail": "https://images-ext-2.discordapp.net/external/ZPxAEUTVk2gIqU8zNqwYlzFCQ8ZTpqt7GcVuOGE0EAk/https/cdn.evergene.io/website/evergene.png?width=1383&height=655",
+    "version": "1.0.0",
+    "color": "#14171a",
+    "tags": [
+        "API",
+        "Reddit-APIs",
+        "Reddit-API",
+        "APIs",
+        "api"
+    ],
```
Comment (SlowLife1661) [inline websites/E/Evergene/dist/metadata.json:19]: Only URL-friendly strings

File: websites/E/Evergene/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Evergene",
+    "author": {
+        "name": "Sound Cat",
+        "id": "723565282844606484"
+    },
+    "category": "other",
+    "logo": "https://cdn.evergene.io/website/evergene-logo.png",
+    "thumbnail": "https://images-ext-2.discordapp.net/external/ZPxAEUTVk2gIqU8zNqwYlzFCQ8ZTpqt7GcVuOGE0EAk/https/cdn.evergene.io/website/evergene.png?width=1383&height=655",
```
Comment (SlowLife1661) [inline websites/E/Evergene/dist/metadata.json:10]: Upload to imgur

**Comment by EncryptedDev** [comment]
Low quality presence?

**Review by Bas950 — CHANGES_REQUESTED** [review]
Kinda low quality presence, could you add more to it?

**Comment by Sound-Cat** [comment]
> Kinda low quality presence, could you add more to it?

I could try, but the website isn't that complex, I want to add some settings

**Comment by ririxidev** [comment]
Heya @Sound-Cat! We have some news for you that you should take look at:
We are so sorry but we need to deny your pull request regarding this presence due of:
 - Website doesn't have a lot of things to show so this site at least for me doesn't qualify to be as presence.
 
Regards,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721.

**Pull request closed without merging**